### PR TITLE
Add PlotNFT Creation Limit.

### PR DESCRIPTION
**Pull request by jack60612**

### CODE REVIEW

**Review by seeden — COMMENTED** [review]
We are using component <Trans> for all localized strings. 
Please update your Alert component

**Review by seeden — APPROVED** [review]

**Review by seeden — APPROVED** [review]

**Comment by mariano54** [comment]
@seeden can we merge this after conflict fix?

**Comment by mariano54** [comment]
Limit increased to 100 in chia-blockchain, but we can set a lower limit in here to be safe:
https://github.com/Chia-Network/chia-blockchain/blob/main/chia/wallet/derive_keys.py#L14

File: packages/api/README.md
Comment (paninaro) [inline packages/api/README.md:50]: Remove please

File: packages/gui/src/components/pool/PoolOverview.tsx
Comment (paninaro) [inline packages/gui/src/components/pool/PoolOverview.tsx:6]: This file has changed in main, so the imports look different. In any event, `Alert` should be imported from `@mui/material`

File: packages/gui/src/components/pool/PoolOverview.tsx
Comment (paninaro) [inline packages/gui/src/components/pool/PoolOverview.tsx:136]: I'm confused by this, and I'm not sure if it's just a variable naming issue or if the logic is wrong.

Is the intent of the name `exceededNFTLimit` as in we've exceeded the NFT limit? Or is it `passed` as in we passed the test that we're below the NFT limit?

From the code below, I suspect it's the former `exceededNFTLimit`, in which case shouldn't the assignment be:

`const exceededNFTLimit = nfts?.length >= 50;`

**Review by paninaro — CHANGES_REQUESTED** [review]
Please see the comment I left re: passedNFTLimit

File: packages/gui/src/components/pool/PoolOverview.tsx
Comment (jack60612) [reply packages/gui/src/components/pool/PoolOverview.tsx:6]: Im going to catch it up to main

File: packages/api/README.md
Comment (jack60612) [reply packages/api/README.md:50]: Oops

File: packages/gui/src/components/pool/PoolOverview.tsx
Comment (jack60612) [reply packages/gui/src/components/pool/PoolOverview.tsx:25]: Sorry it should be exceededNFTLimit, thanks!

**Comment by jack60612** [comment]
Oh wait dont merge!

**Comment by jack60612** [comment]
Ready for review

File: packages/gui/src/components/plotNFT/select/PlotNFTSelectPool.tsx
Comment (paninaro) [inline packages/gui/src/components/plotNFT/select/PlotNFTSelectPool.tsx:177]:
```suggestion
          <Trans>You already have 15 or more Plot NFTs.</Trans>
```

Comment (jack60612) [reply packages/gui/src/components/plotNFT/select/PlotNFTSelectPool.tsx:177]: Oops, meant to change that back to 50

**Review by paninaro — APPROVED** [review]
lgtm

**Pull request merged**
